xtask/src/commands/clippy.rs

```diff
@@ -12,19 +12,57 @@ pub struct Clippy {
 
 impl Run for Clippy {
     fn run(self) -> Result<()> {
-        let mut args = vec![
-            "clippy",
-            "--all-targets",
+        // Arguments that go before feature flags (e.g., "clippy", "--fix")
+        let mut clippy_command = vec!["clippy"];
+        if self.fix {
+            clippy_command.push("--fix");
+        }
+
+        // Define common non-version-specific features for ratatui-crossterm. These will be enabled
+        // for both crossterm 0.28 and 0.29 runs. `underline-color`` is part of default, but with
+        // `--no-default-features``, we must add it.
+        let common_features = [
+            "serde",
+            "underline-color",
+            "scrolling-regions",
+            "unstable",
+            "unstable-backend-writer",
+        ]
+        .join(",");
+
+        let clippy_options = ["--", "-D", "warnings"];
+
+        // Run Clippy on `ratatui-crossterm` with `crossterm_0_28` and `crossterm_0_29`
+        for crossterm_feature in ["crossterm_0_28", "crossterm_0_29"] {
+            let mut command = clippy_command.clone();
+            let features = format!("{common_features},{crossterm_feature}");
+            // Note that adding --tests or --benches causes clippy to pick up the default features.
+            // I'm not sure why this is the case (JM 2025-05-10).
```

Comment:
Any ideas on this?

Reply:
Never ran into it... But intuitively I'd guess that using those flags will use default features...